src/core/evaluator.js

```diff
@@ -4873,8 +4883,26 @@ class EvalState {
     this.ctm = new Float32Array(IDENTITY_MATRIX);
     this.font = null;
     this.textRenderingMode = TextRenderingMode.FILL;
-    this.fillColorSpace = ColorSpace.singletons.gray;
-    this.strokeColorSpace = ColorSpace.singletons.gray;
+    this._fillColorSpace = ColorSpace.singletons.gray;
+    this._fillColorSpacestrokeColorSpace = ColorSpace.singletons.gray;
```

Comment:
Could this typo may explain the test failures?
```suggestion
    this._strokeColorSpace = ColorSpace.singletons.gray;
```